tests/propagation/test_http.py

```diff
@@ -1,42 +1,63 @@
 from unittest import TestCase
-from nose.tools import eq_, ok_
+from nose.tools import eq_
 from tests.test_tracer import get_dummy_tracer
 
-from ddtrace.span import Span
-from ddtrace.context import Context, ThreadLocalContext
-
 from ddtrace.propagation.http import (
     HTTPPropagator,
     HTTP_HEADER_TRACE_ID,
     HTTP_HEADER_PARENT_ID,
     HTTP_HEADER_SAMPLING_PRIORITY,
 )
 
+
 class TestHttpPropagation(TestCase):
     """
     Tests related to the ``Context`` class that hosts the trace for the
     current execution flow.
     """
+
     def test_inject(self):
         tracer = get_dummy_tracer()
 
         with tracer.trace("global_root_span") as span:
+            span.context.sampling_priority = 2
             headers = {}
             propagator = HTTPPropagator()
             propagator.inject(span.context, headers)
 
             eq_(int(headers[HTTP_HEADER_TRACE_ID]), span.trace_id)
             eq_(int(headers[HTTP_HEADER_PARENT_ID]), span.span_id)
-            # TODO: do it for priority too
-
+            eq_(
+                int(headers[HTTP_HEADER_SAMPLING_PRIORITY]),
+                span.context.sampling_priority,
+            )
 
     def test_extract(self):
         tracer = get_dummy_tracer()
 
         headers = {
-            HTTP_HEADER_TRACE_ID: '1234',
-            HTTP_HEADER_PARENT_ID: '5678',
-            HTTP_HEADER_SAMPLING_PRIORITY: '1',
+            HTTP_HEADER_TRACE_ID: "1234",
```

Comment:
should we use `HTTP_X_DATADOG_TRACE_ID` also here?

Reply:
This test is semantically unchanged.

To test the `"HTTP_X_DATADOG_TRACE_ID"` key we have the `test_WSGI_extract` test method below.

Reply:
My point is to not use a variable but a plain string that detects any change in our code. In general the input must not be generated using our code otherwise the test is not reliable (it changes with our code changes so you don't detect if the software is different). It should be `x-datadog-trace-id` or similar.

Reply:
Ohhh, okay. I didn't realize that you wanted me to change the functionality on the existing test. I'll make this change.